### ISSUE TITLE
[Question] Swap EPG info from OSD

### ISSUE THREAD

**Issue by notoco** [jurialmunkey/skin.arctic.horizon.2#796]
### Skin section

PVR and Live TV

### Feature description

Currently, when watching TV and the OSD is displayed - you can use the down arrow to display the EPG with the next channels, and then the EPG with the next programs in the watched channel.
Can these items be swapped?

### Why is this feature needed? Please justify and explain.

It seems more logical that I want to check what's next on a given channel first, and then what's on other channels

### Screenshots and Additional Info

![screenshot00000](https://user-images.githubusercontent.com/36659110/215839166-dc677216-728f-41e3-976f-b7129bdf6603.png)
![screenshot00024](https://user-images.githubusercontent.com/36659110/215839193-b89f5ca7-7177-497d-b7cf-bcbb82f1cb8e.png)
![screenshot00025](https://user-images.githubusercontent.com/36659110/215839199-6ef5d6db-15a4-4ac7-9227-89a941e90759.png)

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Yeah I think this makes a lot of sense, especially now that there's a button right there to view channels in the vertical list too.

Navigation is a bit tricky to handle underneath because the "next programs" epg is actually a separate dialog (I just use some skinning tricks to make it look like it's part of the osd like the "next channels" are) - - but I should be able to work something out.

**Comment by notoco** [comment]
Btw. Any chance to add a other font? Amazon for example (on my screenshot you can see it - but there os problem with long title wraping)

**Comment by notoco** [comment]
> Yeah I think this makes a lot of sense, especially now that there's a button right there to view channels in the vertical list too.
> 
> Navigation is a bit tricky to handle underneath because the "next programs" epg is actually a separate dialog (I just use some skinning tricks to make it look like it's part of the osd like the "next channels" are) - - but I should be able to work something out.

Any chance?
